src/main/java/com/example/TicketFinder/ObjectMapperConfiguration.java

```diff
@@ -1,4 +1,4 @@
-package com.example.demo;
+package com.example.TicketFinder;
```

Comment:
Package names should be lower case, please check [here](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) 

Also, an example is unnecessary, please rename the package name to `com.ticketfinder`